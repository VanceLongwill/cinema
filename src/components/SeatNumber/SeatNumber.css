.SeatNumber {
  font-weight: 700;
  padding: var(--spacing-sm);
}
