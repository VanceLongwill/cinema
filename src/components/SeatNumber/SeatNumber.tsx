import React from "react";

import "./SeatNumber.css";

export const SeatNumber: React.FunctionComponent<{ value: string }> = ({
  value
}) => <span className="SeatNumber">{value}</span>;
