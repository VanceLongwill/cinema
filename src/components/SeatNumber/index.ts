import { SeatNumber } from "./SeatNumber";
export default SeatNumber;
