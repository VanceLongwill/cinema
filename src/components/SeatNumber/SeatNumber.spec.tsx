import * as React from "react"

import { mount } from "enzyme"

import { SeatNumber } from "./SeatNumber"

describe("SeatNumber", () => {
  it("should render without errors", () => {
    expect(mount(<SeatNumber />)).toMatchSnapshot()
  })
})
