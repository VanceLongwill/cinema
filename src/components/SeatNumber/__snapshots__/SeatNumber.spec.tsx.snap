// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeatNumber should render without errors 1`] = `
<SeatNumber>
  <span
    className="SeatNumber"
  />
</SeatNumber>
`;
